orion/sites/admin/src/location/location.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+require('./location.styl');
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router-dom';
+
+export class LocationPage extends React.Component {
+    render() {
+        var numLocations = 5;
+        var fakeLocation = {
+            locId: 123,
+            address: "123 Sesame Street",
+            roomNum: 23
+        }
+        return (
+            <div id="view-location">
+                <h1>All Locations ({numLocations})</h1>
+                <ul id="list-heading">
+                    <li className="li-med">Location ID</li>
+                    <li className="li-med">Address</li>
+                    <li className="li-med">Room Number</li>
+                </ul>
+                <ul>
+                    <LocationRow locationObj = {fakeLocation}/>
```

Comment:
Can you try 2-3 more `LocationRows`? It would be good to see if multiple rows look and act as intended.